src/services/web-monitoring-api.js

```diff
@@ -77,6 +77,82 @@ export default class WebMonitoringApi {
             ));
     }
 
+    /**
+     * Add change information to the dictionary of insignificant changes in
+     * Google sheets.
+     *
+     * @param {Page}    page
+     * @param {Version} fromVersion
+     * @param {Version} toVersion
+     * @param {Object}  annotation
+     * @returns {Promise<boolean>}
+     */
+    addChangeToDictionary (page, fromVersion, toVersion, annotation) {
```

Comment:
Probably not worth the effort because we have only the 2 use cases and they're both temporary in the long run, but curious to know for my own benefit, why wouldn't we refactor the 2 methods below into 1 with additional parameters for postURL and errorMessage (or constructing an error message from postURL)?